src/components/Reports/ReportEdit.tsx

```diff
@@ -196,13 +227,40 @@ export default function ReportEdit(): JSX.Element {
           additionalRightButtons={rightButtons}
         >
           {report &&
+            draftReport &&
             (showAnnual ? (
               <ReportFormAnnual editable={true} report={report} />
             ) : (
-              <ReportForm editable={true} report={report} onUpdateReport={updateReport} />
+              <ReportForm
+                editable={true}
+                draftReport={draftReport}
+                onUpdateReport={updateReport}
+                allSeedbanks={getAllSeedBanks(organization)
+                  .filter((f) => !!f)
+                  .map((f) => seedbankFromFacility(f!, report))}
+                onUpdateSeedbank={updateSeedbank}
+                onUpdateSeedbankWorkers={updateSeedbankWorkers}
+              />
             ))}
         </PageForm>
       )}
     </TfMain>
   );
 }
+
+const seedbankFromFacility = (facility: Facility, originalReport: Report): GetSeedBankV1 => {
+  const existingSeedbank = originalReport.seedBanks?.find((sb) => sb.id === facility.id);
+  if (existingSeedbank) {
+    return existingSeedbank;
+  }
+
+  return {
+    id: facility.id,
+    name: facility.name,
+    buildCompletedDateEditable: true,
+    buildStartedDateEditable: true,
+    operationStartedDateEditable: true,
+    totalSeedsStored: 0,
+    workers: {},
+  };
```

Comment:
could move this as a util func into report service?

Reply:
Yes, good call